### PR TITLE
Prose: Chapter 52 — Bidirectional Context (#394)

### PR DESCRIPTION
**Pull request by krisztiankoos**
## Summary

Publishes the first prose draft for AI History Chapter 52, Bidirectional Context.

- Adds the reader-facing BERT chapter at 4,001 words, inside the approved 4,000-5,000 range and below the 5,000-word cap.
- Marks the Ch52 research contract as prose_review while awaiting cross-family prose/source review.
- Links Ch52 from the AI History index; publication count remains 51 until prose review acceptance.

## Source discipline

Drafted only from the dual-cleared Ch52 contract: Devlin et al. 2018 BERT paper, Google Research BERT release post, and GitHub API metadata for google-research/bert. No invented dollar costs, adoption scenes, or unsupported comprehension claims.

## Checks

- npm run astro -- sync
- git diff --check
- wc -w src/content/docs/ai-history/ch-52-bidirectional-context.md -> 4001

## Review plan

Requesting cross-family source/prose review from Claude and Gemini. Gemini should use gemini-3.1-pro-preview, falling back to gemini-3-flash-preview if unavailable.

### CODE REVIEW

**Comment by krisztiankoos** [comment]
Cross-family source/prose review (Claude): APPROVE.

Claude reviewed PR #565 against the approved Ch52 contract and found no must-fix issues. Source checks covered WordPiece/30k vocabulary, 15 percent MLM masking with 80/10/10 replacement, NSP framing, BooksCorpus/Wikipedia data, BERTBASE/BERTLARGE sizes, Cloud TPU pre-training hardware/time, fine-tuning time claims, benchmark framing, and google-research/bert repo metadata. Boundary checks passed: no invented dollar costs, no invented adoption scenes, no human-comprehension overclaim, no unsupported WordPiece/OOV claim, and no unsupported community adoption claim.

Gemini review was requested with gemini-3.1-pro-preview and automatically fell back to gemini-3-flash-preview, but the fallback hit Gemini rate limits. Claude provides the required independent-family review for this Codex-authored prose PR.

Bridge results: Claude message #3109; Gemini unavailable/rate-limited after message #3108.
